composer.lock

```diff
@@ -7893,116 +7893,6 @@
             ],
             "time": "2018-04-30T19:57:29+00:00"
         },
-        {
-            "name": "symfony/polyfill-ctype",
```

Comment:
normal this deletion?

Reply:
Yes I think it's because someone ran composer update with his local composer instead of the one provided by docker.